python/python/lancedb/table.py

```diff
@@ -893,6 +896,27 @@ def compact_files(self, *args, **kwargs):
         For most cases, the default should be fine.
         """
 
+    @abstractmethod
+    def optimize(
+        self,
+        *,
+        cleanup_older_than: Optional[timedelta] = None,
+        delete_unverified: bool = False,
+    ):
+        """
+        Optimize the indices of the table.
+
+        This incrementally updates the indices of the table,
+        to make queries faster.
+
+        Note: This function is not available in LanceDb Cloud (since LanceDb
+        Cloud manages index optimization for you automatically)
+
+        Arguments are passed onto
+        :meth:`lance.dataset.DatasetOptimizer.optimize_indices`.
+        For most cases, the default should be fine.
```

Comment:
Is this true?  I don't see any `kwargs` being passed on.

Reply:
Oh I think it's from `optimize_indices`, fixed

Reply:
Oh I think it's from optimize_indices, fixed